src/scenes/game-scene.ts

```diff
@@ -46,29 +48,61 @@ const levelConfig = {
   ],
 };
 
+const soundConfig = [
+  {
+    key: 'punch',
+    filePath: 'audio/sfx/punch.wav',
+    loop: false
+  },
+  {
+    key: 'music1',
+    filePath: 'audio/music/Fluffing-a-duck.mp3',
```

Comment:
might be a good idea to record the source for the mp3s in the README?

i should also add the source of the parallaxes (that is "parallax" search on itch.io)